### PR TITLE
[update] Modify basic parameters

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

https://github.com/timescale/docs/pull/1074#issuecomment-1134678472)

# Links

Fixes #[insert issue link, if any]

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] Have you checked the built version of this content?

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

File: cloud/service-settings/customize-configuration.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ an editable field to apply your change. Clicking anywhere outside of that field
 saves the value to be applied.
 
 <highlight type="note">
-The range for some of the parameters vary depending on the plan. For example,
-the range for `max_connections` is 0 to 25 in the smallest compute tier, but 25
-to 500 in the next tier.
+The values for some of the parameters vary depending on the plan. For example,
+the value for `max_connections` is exactly 25 in the smallest compute tier, but
+ranges from 25 to 500 in the next tier.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply cloud/service-settings/customize-configuration.md:38]: What about "higher tier plans" or something similar? This wording makes me feel like it should be followed by a colon and a list 😉

[the remark this answers is not in the document]
Comment (lukas-bernert) [reply cloud/service-settings/customize-configuration.md:38]: Sure, whatever you think sounds best. But the current wording makes me think "what about the ranges in the tiers after that?".

[the remark this answers is not in the document]
Comment (Loquacity) [reply cloud/service-settings/customize-configuration.md:38]: Oh, I absolutely agree! Just trying to figure out the best way to say that accurately :)

[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply cloud/service-settings/customize-configuration.md:38]: @Loquacity  and @lukas-bernert  How about : "25 to 500 in higher compute tiers"

[the remark this answers is not in the document]
Comment (lukas-bernert) [reply cloud/service-settings/customize-configuration.md:38]: Perfect 👍

**Review by lukas-bernert — APPROVED** [review]

**Pull request merged**